### DISCUSSION TITLE
5.2 error when creating codespace (even with 8-core)

### DISCUSSION THREAD

**Discussion by jenw-ua** [ua-gist604b-f24/syllabus#17] [General]
Anyone else still having a problem with the 5.2 module? I can't get past trying to create a codespace with 8-core. I'm still getting this error...
<img width="1538" alt="Screenshot 2024-11-14 at 10 18 36 AM" src="https://github.com/user-attachments/assets/355db711-bafe-4dab-88a5-54b05022784f">

**Comment by kev-bone** [comment]
I haven't started, but I was able to build a codespace just now. He updated the Dockerfile in the dev container a few days ago, so you might need to reaccept the assignment from the syllabus schedule.

**Comment by vmwielgus** [comment]
Were you able to get this to work? Is there another way to reaccept the assignment other than clicking on it from the syllabus schedule? I've tried reopening the assignment multiple times from there, switching from PC to Mac, still can't complete the assignment and now I'm behind.

**Comment by jenw-ua** [comment]
Module 5 I was able to re accept after I figured out it was rereleased. I was not able to fully complete any of the module 6 assignments due to many errors in each of them. It really is unfortunate because for the amount of time that I spent on them and to not be able to complete them or learn them...I hope it worked out for you though!  Good luck to us all.

**Reply by vmwielgus** [reply]
Turns out I'd accepted the assignment before the dev container was updated, but Dr. O synced my fork and I was finally able to complete 5-2. I'm attempting Module 6 now....
